README.org

```diff
@@ -23,10 +23,19 @@
   - Create a ~.lein/profiles.clj~ file in your home directory.
     + The contents of the file should be as follows:
       #+begin_src clojure
-        {:repl {:plugins [[cider/cider-nrepl "0.14.0"]
-                          [refactor-nrepl "2.2.0"]]
+        {:repl {:plugins [[cider/cider-nrepl "0.16.0"]]
+
+                          ;; use stable version when this is released:
+                          ;; https://github.com/clojure-emacs/refactor-nrepl/issues/221
+                          [refactor-nrepl "2.4.0-SNAPSHOT"]]
                 :dependencies [^:replace [org.clojure/tools.nrepl "0.2.12"]]}}
       #+end_src
+    + Additionally, since for the time being, we are using a snapshot
+      version of refactor-nrepl, you might have to set the following
+      environment var:
+      #+BEGIN_SRC shell
+      export LEIN_JVM_OPTS=-Dhttps.protocols=TLSv1.2
+      #+END_SRC
     + You don't need the cider-nrepl or refactor-nrepl plugin section
```

Comment:
Let's wait a few days for the 2.4.0 release. Would prefer to merge this change in with that. If it doesn't get created upstream by then, then lets drop to 2.3.1